public/app/screens/main/sections/devices/room/room.jsx

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 var React = require('react');
-var Device = require('screens/main/devices/device/index');
+var Device = require('../device');
+
+require('./style.scss');
```

Comment:
was not part of the PR, we worked on the main screen and navigation bar. This could be done by a separate PR